### ISSUE TITLE
code of loss function

### ISSUE THREAD

**Issue by DanMerry** [betterzhou/Open_World_ECG_Classification#1]
# Thank you very much for sharing the code. During the training process, I encountered an issue: regardless of whether I use CELoss or SupConLoss, both losses use features and logits as inputs for the loss calculation. However, in contrastive learning, the concepts of anchor and target should be different. How can I distinguish between the usage of these two losses?
![image](https://github.com/user-attachments/assets/0db8848f-038c-49e9-8e74-e4a2cf1dec4f)

**Comment by betterzhou** [comment]
Hi, Merry, glad to hear from you. 
(1) The code supports two different training modes. Hence, you can assign either CELoss or SupConLoss. We used SupConLoss in the proposed method.
(2) You may check the details of the SupConLoss in loss_func.py.
